pbsync/pbsync.py

```diff
@@ -13,7 +13,7 @@
 from pbpy import pbpy_version
 from pbpy import pbdispatch
 
-import pbsync_version
+from pbsync import pbsync_version
```

Comment:
Our debugging & development environment is Visual Studio code, and this change breaks debugger launch configurations. Please revert.